src/pipeline/physics_pipeline.rs

```diff
@@ -937,4 +939,68 @@ mod test {
         // Expect body to now be in active_dynamic_set
         assert!(islands.active_dynamic_set.contains(&h));
     }
+
+    #[test]
+    fn joint_step_delta_time_0() {
+        let mut colliders = ColliderSet::new();
+        let mut impulse_joints = ImpulseJointSet::new();
+        let mut multibody_joints = MultibodyJointSet::new();
+        let mut pipeline = PhysicsPipeline::new();
+        let mut bf = BroadPhaseMultiSap::new();
+        let mut nf = NarrowPhase::new();
+        let mut islands = IslandManager::new();
+
+        let mut bodies = RigidBodySet::new();
+
+        // Initialize bodies
+        let rb = RigidBodyBuilder::fixed().additional_mass(1.0).build();
+        let h = bodies.insert(rb.clone());
+        let rb_dynamic = RigidBodyBuilder::dynamic().additional_mass(1.0).build();
+        let h_dynamic = bodies.insert(rb_dynamic.clone());
+
+        // Add joint
+        #[cfg(feature = "dim2")]
+        let joint = RevoluteJointBuilder::new()
+            .local_anchor1(point![0.0, 1.0].into())
+            .local_anchor2(point![0.0, -3.0].into());
```

Comment:
What is the `.into()` for?

Reply:
for transforming into dimension3 I believe ? or a leftover from f32 / f64 🤔  : I'll double check.